scripts/chassis_lag_id_checker.py

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env python3
```

Comment:
is there a plan to extend this to other DBs consistency check as well in future? if so, do you want to instead call it as chassis_db_consistency_checker?